recipe/meta.yaml

```diff
@@ -81,7 +81,7 @@ requirements:
 test:
   requires:
     - pip
-    - hypothesis >=5.5.3
+    - hypothesis >=6.13.0
```

Comment:
Seems like it still `hypothesis >=5.5.3` for **pandas 1.5.3**, see https://github.com/pandas-dev/pandas/blob/2e218d10984e9919f0296931d92ea851c6a6faf5/setup.cfg#L55

Reply:
Developers suggest >=6.13.0 here: https://pandas.pydata.org/docs/getting_started/install.html#running-the-test-suite. Added a comment to the recipe